test-files/io/ServiceOsmApiDbBulkInserterTest/psql-offline-large-starting-ids.sql

```diff
@@ -141,11 +141,12 @@ COPY current_relation_members (relation_id, member_type, member_id, member_role,
 
 COPY current_relation_tags (relation_id, k, v) FROM stdin;
 3000000005	building	yes
+3000000005	type	multipolygon
```

Comment:
More timestamps to ignore, with the `type` being inserted here and below in the `relation_tags` table.